client/components/resizable-iframe/index.jsx

```diff
@@ -97,6 +97,22 @@ export default React.createClass( {
 					subtree: true
 				} );
 
+				// Hack: Remove viewport unit styles, as these are relative
+				// the iframe root and interfere with our mechanism for
+				// determining the unconstrained page bounds.
+				function removeViewportStyles( ruleOrNode ) {
+					[ 'width', 'height', 'minHeight', 'maxHeight' ].forEach( function( style ) {
+						if ( /^\\d+(vmin|vmax|vh|vw)$/.test( ruleOrNode.style[ style ] ) ) {
+							ruleOrNode.style[ style ] = '';
+						}
+					} );
+				}
+
+				[].forEach.call( document.querySelectorAll( '[style]' ), removeViewportStyles );
```

Comment:
Array.prototype.forEach.call would avoid allocating three arrays for no purpose. :)

Or use lodash.